Azure/azure-key-vault/ingest/parser.yml

```
@@ -0,0 +1,33 @@
+name: azure-key-vault
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: "database"
+          event.dataset: "keyvault"
+          event.outcome: "{{json_event.message.resultSignature}}"
+          event.action: "{{json_event.message.operationName}}"
+          event.type: ["access", "change", "info"]
```

I suggest using only one type:
```suggestion
          event.type: ["access"]
```